pkgs/development/python-modules/filecheck/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+}:
+
+buildPythonApplication rec {
+  pname = "filecheck";
+  version = "0.0.22";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-Ae4ye2t0LO0v2eRDGdLXBqRXPl4O5yYZONDeFqd+XAY=";
+  };
+
```

Comment:
Missing pythonImportsCheck

Reply:
The pypi source does not package the tests.

Reply:
If that's the case, please use fetchFromGitHub instead of fetchPypi.

Reply:
I would strongly prefer not to, the github repo also uses poetry instead of setuptools.

Reply:
Changing to github source is a standard procedure to get tests executed which we should always do if they work in the sandbox.

Building from poetry is easy. Add ``format = "pyproject";`` and check if the project uses poetry or poetry-core and add that to nativeBuildInputs.